media-gfx/waifu2x-ncnn-vulkan/waifu2x-ncnn-vulkan-20210521.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+NCNN_VERSION="20210525" # pin near bundled copy until more stable
```

Comment:
You use it only once and I rather not have global variables defined here. Can you please just specify it in DEPEND?